### PR TITLE
Document various parts of the source tree

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: include/constants/quest_log.h [hunk 1]
```diff
@@ -2,9 +2,14 @@
 #define GUARD_CONSTANTS_QUEST_LOG_H
 
 // TODO: Name and use state constants
```
Comment (jiangzhengwenjz) [inline include/constants/quest_log.h:4]: remove the comment

File: include/help_message.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#ifndef GUARD_HELP_MESSAGE_H
+#define GUARD_HELP_MESSAGE_H
+
```
Comment (jiangzhengwenjz) [inline include/help_message.h:3]: `#include "global.h"`

File: include/scrcmd.h [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #ifndef GUARD_SCRCMD_H
 #define GUARD_SCRCMD_H
```
Comment (jiangzhengwenjz) [inline include/scrcmd.h:3]: Can this header be removed?

File: src/trade.c [hunk 4]
```diff
@@ -26,34 +26,34 @@
...
     /*0x0002*/ u8 filler_2[0x28 - 2];
     /*0x0028*/ u8 partyIcons[2][PARTY_SIZE];
     /*0x0034*/ u8 tradeMenuCursorSpriteIdx;
     /*0x0035*/ u8 tradeMenuCursorPosition;
     /*0x0036*/ u8 partyCounts[2];
     /*0x0038*/ bool8 tradeMenuOptionsActive[13];
-    /*0x0045*/ u8 unk_45[2][PARTY_SIZE];
-    /*0x0051*/ u8 unk_51[2][PARTY_SIZE];
+    /*0x0045*/ u8 battleableFlags[2][PARTY_SIZE];
+    /*0x0051*/ u8 eggFlags[2][PARTY_SIZE];
```
Comment (jiangzhengwenjz) [inline src/trade.c:38]: `bool8`

File: src/trade.c [hunk 5]
```diff
@@ -1359,90 +1359,90 @@ static bool8 shedinja_maker_maybe(void)
...
         {
             sub_800A474(4);
         }
-        sTradeMenuResourcesPtr->unk_69++;
+        sTradeMenuResourcesPtr->state++;
         break;
     case 20:
         if (GetBlockReceivedStatus() == 3)
         {
             Trade_Memcpy(sTradeMenuResourcesPtr->unk_A9, gBlockRecvBuffer[id ^ 1], 11);
```
Comment (jiangzhengwenjz) [inline src/trade.c:1443]: `sizeof(sTradeMenuResourcesPtr->unk_A9)`

File: src/slot_machine.c [hunk 6]
```diff
@@ -2097,26 +2212,26 @@ static bool8 sub_81416C8(u8 * state, struct SlotMachineSetupTaskData * ptr)
         SetGpuReg(REG_OFFSET_WIN1H, 0xA0);
```
Comment (jiangzhengwenjz) [inline src/slot_machine.c:2212]: `WIN_RANGE`

File: src/slot_machine.c [hunk 7]
```diff
@@ -1843,53 +1958,53 @@ static bool8 sub_8141198(u8 * state, struct SlotMachineSetupTaskData * ptr)
         SetGpuReg(REG_OFFSET_DISPCNT, 0);
         ResetBgPositions();
         ResetBgsAndClearDma3BusyFlags(0);
```
Comment (jiangzhengwenjz) [inline src/slot_machine.c:1960]: `FALSE`

File: include/help_message.h (hunk 2 above)
Comment (PikalaxALT) [reply include/help_message.h:3]: Not necessary

File: src/trade.c [hunk 8]
```diff
@@ -1440,7 +1440,7 @@ static bool8 shedinja_maker_maybe(void)
     case 20:
         if (GetBlockReceivedStatus() == 3)
         {
-            Trade_Memcpy(sTradeMenuResourcesPtr->unk_A9, gBlockRecvBuffer[id ^ 1], 11);
+            Trade_Memcpy(sTradeMenuResourcesPtr->giftRibbons, gBlockRecvBuffer[id ^ 1], 11);
```
Comment (jiangzhengwenjz) [inline src/trade.c:1443]: sizeof

File: include/link_rfu.h [hunk 9]
```diff
@@ -139,7 +139,7 @@ typedef struct UnkRfuStruct_2
     /* 0x0e4 */ u8 cmd5f00Ack[5];
     /* 0x0e9 */ u8 cmd_6600_recvd[5];
     /* 0x0ee */ vu8 errorState;
-    /* 0x0ef */ u8 unk_ef;
+    /* 0x0ef */ u8 isShuttingDown;
```
Comment (jiangzhengwenjz) [inline include/link_rfu.h:142]: `bool8`

File: src/trade.c [hunk 10]
```diff
@@ -54,7 +56,7 @@ struct TradeMenuResources
     /*0x007F*/ u8 filler_7F;
```
Comment (jiangzhengwenjz) [inline src/trade.c:56]: `filler_71` and this can be removed

**Pull request merged**